datajunction-server/datajunction_server/internal/validation.py

```diff
@@ -128,7 +129,12 @@ async def validate_node_data(  # pylint: disable=too-many-locals,too-many-statem
                 name=column_name,
                 display_name=labelize(column_name),
                 type=column_type,
-                attributes=existing_column.attributes if existing_column else [],
+                attributes=[
```

Comment:
This was an annoying issue: the `attributes` field on the `Column` database object is actually a `ColumnAttribute` object and not an `Attribute` object. Since `ColumnAttribute` is the relational link between an `Attribute` and a `Column`, we actually need to recreate this for any copied columns. Otherwise, it'll cause problems downstream since it may reference a column that doesn't exist.